.github/workflows/generate-desktop-targets.yml

```diff
@@ -19,7 +17,7 @@ defaults:
     shell: bash
 
 env:
-  FLEET_DESKTOP_VERSION: 1.40.1
+  FLEET_DESKTOP_VERSION_FALLBACK: 1.40.1
```

Comment:
IIRC we have never released desktop or orbit separately.

They are separate for the one reason that we support fleetd without Fleet Desktop and because they sort of need to be separate binaries (because they run in different contexts, root vs user).

So, you don't need to cover the scenario where one is released and not the other. They are released together.

Reply:
This has "workflow_dispatch" on it, so someone can technically run it at any time.  Should I bail out if it detects that it's being run on a branch that doesn't have a tag?

Reply:
Maybe remove `workflow_dispatch` for now? (It was probably only needed during development of the automation?)

Reply:
done!